### ISSUE TITLE
SSL CA cert error when cargo fetches index

### ISSUE THREAD

**Issue by janderholm** [meta-rust/meta-rust#246]
Cargo cannot fetch crates.io index when trying to compile a package that has been checked out using `devtool modify`.

## Version(s) of meta-rust
master (c5ff05d)

## Version(s) of poky and/or oe-core
thud

## Expected result
cargo should be able to fetch index

## Actual result

```
| warning: spurious network error (2 tries remaining): [77] Problem with the SSL CA cert (path? access rights?); class=Net (12)
| warning: spurious network error (1 tries remaining): [77] Problem with the SSL CA cert (path? access rights?); class=Net (12)
```

**Comment by srwalter** [comment]
Issue should be fixed by #247
